gallery/src/components/NxSearchDropdown/NxSearchDropdownPage.tsx

```diff
@@ -169,7 +169,7 @@ const NxSearchDropdownPage = () =>
         <NxP>
           The search functionality of <NxCode>NxSearchDropdown</NxCode> should be used with a debounce in order
           to prevent excessive queries to the backend. The standard timing value to use for that debounce is provided
-          via the <NxCode>NX_SEARCH_DROPDOWN_DEBOUNCE_TIME</NxCode> export. See
+          via the <NxCode>NX_STANDARD_DEBOUNCE_TIME</NxCode> export. See
```

Comment:
We should mention the deprecated name here too.

Reply:
Updated here d5853d6eddf2149c17482c63ff3736de7a3d6329.